package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wikia/react-common",
-  "version": "11.1.0",
+  "version": "11.2.0",
```

Comment:
Don't do this; this is updated automatically when releasing

Reply:
It snuck in there. I removed. You reviewed to quickly. 😄

Reply:
🥇